### DISCUSSION TITLE
Can I use Tailwind css for React Native projects?

### DISCUSSION THREAD

**Discussion by umair-mirza** [tailwindlabs/tailwindcss#2913] [General]
I was wondering if Tailwind css can be used to design React Native applications entirely, without the need for any other UI kit? Liek for example, If I were to develop a full-fledged e-commerce application on React Native, would I be able to design it completely using Tailwind css?

**Comment by sven09** [comment]
Hi,

maybe this helps: https://github.com/vadimdemedes/tailwind-rn
